episodes/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  nixpkgs ? builtins.getFlake "nixpkgs",
+  system ? builtins.currentSystem,
+  pkgs ? nixpkgs.legacyPackages.${system},
+  lib ? nixpkgs.lib,
+}: let
+  episodes = lib.pipe ./. [
+    builtins.readDir
+    # filter directories
+    (elems: builtins.filter (name: elems.${name} == "directory") (builtins.attrNames elems))
+    # parse all ints in names and sort by them
+    (builtins.sort (l: r: (lib.toIntBase10 l) < (lib.toIntBase10 r)))
```

Comment:
I'd say let's pad the filenames with 0's, so 001, 002, .., 010, 011, ...

Then this code isn't needed, and also the file listing shows the correct order when looking at it